zeek

```diff
@@ -0,0 +1,159 @@
+#!/bin/bash
+#Sample start/stop script for Zeek running inside docker
+#based on service_script_template v0.2
+#Many thanks to Logan for his Active-Flow init script, from which some of the following was copied.
+#V0.3.2
+
+if [ -n "$1" -a -z "$2" ]; then
+	case "$1" in
+	start|stop|restart|force-restart|status|reload|enable|disable|pull|update)
+		action="$1"
+		;;
+	*)
+		echo "Unrecognized action $1 , exiting" >&2
+		exit 1
+		;;
+	esac
+else
+	echo 'This script expects a single command line option (start, stop, restart, status, reload, enable or disable).  Please run again.  Exiting' >&2
+	exit 1
+fi
+
+host_zeek='/usr/local/zeek'
+
+host_zeek_logs="$host_zeek/logs"
+host_zeek_spool="$host_zeek/spool"
+host_zeek_etc="$host_zeek/etc"
+host_zeek_node_cfg="$host_zeek_etc/node.cfg"
+
+CONTAINER_NAME="zeek"
+#Note, we force the lts release for stability
+IMAGE_NAME="activecm/zeek:lts"
```

Comment:
Thinking ahead, maybe we should pin this to a more specific tag like `3.0`? The `lts` tag here will match what Zeek considers their current LTS. When Zeek releases 4.0 I expect that will become their new LTS and we may need time to update our projects elsewhere before switching to 4.0.

Reply:
I think we get enough protection by sticking with lts.  If we force 3.0 and they release a 3.0.2 (or something else >3.0), then we're locking the user down to an old version and possibly stopping them from getting security fixes.  Unlike, say, Mongo where the version number can have a direct impact on compatibility, this is largely an independent project whose sole job is generating log files.  As long as it continues to do that, we're good even in the 4.0 upgrade.  I'd put in two thumbs up for leaving it at lts as a good balance between stability and getting security fixes.